libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -236,8 +236,8 @@ string IRGenerator::generate(
 
 	solAssert(_contract.annotation().creationCallGraph->get() != nullptr, "");
 	solAssert(_contract.annotation().deployedCallGraph->get() != nullptr, "");
-	verifyCallGraph(collectReachableCallables(**_contract.annotation().creationCallGraph), move(creationFunctionList));
-	verifyCallGraph(collectReachableCallables(**_contract.annotation().deployedCallGraph), move(deployedFunctionList));
+	verifyCallGraph(collectReachableCallables(**_contract.annotation().creationCallGraph), std::move(creationFunctionList));
+	verifyCallGraph(collectReachableCallables(**_contract.annotation().deployedCallGraph), std::move(deployedFunctionList));
```

Comment:
```suggestion
	verifyCallGraph(
	     collectReachableCallables(**_contract.annotation().creationCallGraph),
	     std::move(creationFunctionList)
	);
	verifyCallGraph(
	     collectReachableCallables(**_contract.annotation().deployedCallGraph), 
             std::move(deployedFunctionList)
        );
```
Just a coding style change.